pkgs/racket-doc/scribblings/foreign/define.scrbl

```diff
@@ -45,6 +46,12 @@ The other options support further wrapping and configuration:
        @racket[_id] to report an error when it is applied if
        @racket[_c-id] was not found in the foreign library.}
 
+ @item{If provided, the @racket[#:make-c-id] option changes
+      the default behavior of @racket[_c-id] using an @tech{ffi
+       identifier convention}, such as converting hyphens to
+      underscores or camel case.
+      Several conventions are provided by
+      @racketmodname[ffi/unsafe/define/conventions].}
```

Comment:
Hmm...I'm torn here. I like most of some of this, but I also want to say at a high level what a convention is here.